t/ssl.t

```diff
@@ -8,7 +8,7 @@ use t::TestCore;
 
 repeat_each(2);
 
-plan tests => repeat_each() * (blocks() * 6 + 1);
+plan tests => repeat_each() * 148;
```

Comment:
I think the old style is easier to maintain.

Reply:
Reverted it.